Issue2885 fan coil unit controls closed loop validation

karthikeyad-pnnl opened the pull request:
@JayHuLBL This is to address the inline comments on #3627. The closed-loop validation model has been added at Buildings.Examples.HydronicSystems.FanCoilUnit. Please review it when you get a chance. Thanks!

JayHuLBL commented:
@karthikeyad-pnnl Would you please address the conflicts? I can then review the PR.
You would need to firstly merge the branch `lbl-srg:issue2885_FanCoilUnitControls` to your branch.

karthikeyad-pnnl commented:
@JayHuLBL Not sure what was causing the merge conflict, since I didn't get any on my branch when I merged the base branch. It seems to be fixed now

JayHuLBL commented:
@karthikeyad-pnnl Thanks! Now the conflicts are gone. I will review it today and tomorrow.

JayHuLBL commented on Buildings/Fluid/ZoneEquipment/package.mo:
```diff
@@ -0,0 +1,41 @@
+within Buildings.Fluid;
+package ZoneEquipment "Package with zone equipment models"
```
empty package.

JayHuLBL commented on Buildings/Resources/Data/Fluid/ZoneEquipment/FanCoilUnit/FanCoilAutoSize_ConstantFanVariableFlow.idf:
```diff
@@ -0,0 +1,2121 @@
+!-Generator IDFEditor 1.51
```
Has the files in package `Resources/Data/Fluid/ZonEquipment` been used any where?

I don't see the folder `Buildings.Fluid.ZoneEquipment`.

JayHuLBL commented on Buildings/Examples/HydronicSystems/BaseClasses/FanCoilUnit/Types/HeaSou.mo:
```diff
@@ -0,0 +1,28 @@
+within Buildings.Examples.HydronicSystems.BaseClasses.FanCoilUnit.Types;
+type HeaSou = enumeration(
+    ele
```
remove line change

JayHuLBL reviewed (changes requested):
@karthikeyad-pnnl See inline comments.

karthikeyad-pnnl commented:
@JayHuLBL I have addressed the comments, and re-verified the files by running the unit tests and the verifyFiles script. Please review it when you get a chance. Thanks!
